prism_compile.c

```diff
@@ -6121,7 +6125,6 @@ pm_compile_node(rb_iseq_t *iseq, const pm_node_t *node, LINK_ANCHOR *const ret,
             ISEQ_COMPILE_DATA(iseq)->last_line = body->location.code_location.end_pos.lineno;
 
             /* wide range catch handler must put at last */
-            ISEQ_COMPILE_DATA(iseq)->catch_except_p = true;
```

Comment:
I don't know if this is correct but it isn't present in the upstream compiler and tests pass without it 🤷🏼‍♀️ it also removes the extra catch table that's not expected on these instructions.